BUILDING.md

```diff
@@ -0,0 +1,39 @@
+# Building Protocol Buffers
+
+These are instructions for building the open source release of protobuf from source.
+
+Given the right tools, this project can be built on Linux, Mac OS X, or Windows though Linux is by far the most commonly used and best tested platform.
+
+## Prerequisites
+
+Likely incomplete
+
+* git
+* JDK 8 or JDK 11. JDK 17 probably doesn't work.
```

Comment:
rm JDK17 line, Bazel works with JDK11 as the host JDK

Reply:
done